tests/test_attenuator.py

```diff
@@ -156,7 +153,7 @@ def test_attenuator_staging(fake_att):
 
 def test_attenuator_third_harmonic():
     logger.debug('test_attenuator_third_harmonic')
-    att = Attenuator('TRD:ATT', MAX_FILTERS-1, name='third', use_3rd=True)
+    att = Attenuator('TRD:ATT', MAX_FILTERS-1, name='third')
     att.wait_for_connection()
```

Comment:
I think this last test can just be removed- it no longer serves any purpose with the 3rd harmonic attenuator positioners gone

Reply:
Did not remove this but updated it to just test the Attenuator in general (0bb3c0b )

Reply:
fair enough!